src/broken_spec/views/search_criteria/SearchCriteriaView_spec.js

```diff
@@ -103,17 +103,6 @@ describe('Search Criteria View', function () {
             view = new SearchCriteriaView({model: searchParameters, searchResultsCollection: resultsCollection});
         });
 
-        it('updates the search term field and triggers a new search when an example term is selected', function () {
```

Comment:
Though this is in the "broken" specs, removed this test as I removed the event, as it's no longer being used.